src/main/kotlin/com/exactpro/th2/read/file/common/AbstractFileReader.kt

```diff
@@ -771,6 +795,11 @@ abstract class AbstractFileReader<T : AutoCloseable>(
     companion object {
         private val LOGGER = KotlinLogging.logger { }
 
+        const val MESSAGE_STATUS_PROPERTY = "STATUS"
+        const val MESSAGE_STATUS_SINGLE = "SINGLE"
+        const val MESSAGE_STATUS_FIRST = "START"
+        const val MESSAGE_STATUS_LAST = "FIN"
```

Comment:
Let's make them lower case. It is more readable in my oppinion
```suggestion
        const val MESSAGE_STATUS_SINGLE = "single"
        const val MESSAGE_STATUS_FIRST = "start"
        const val MESSAGE_STATUS_LAST = "fin"
```